rust/lancedb/Cargo.toml

```diff
@@ -46,8 +46,13 @@ tempfile = "3.5.0"
 rand = { version = "0.8.3", features = ["small_rng"] }
 uuid = { version = "1.7.0", features = ["v4"] }
 walkdir = "2"
+# For integration tests (dev deps aren't allowed to be optional atm)
+aws-sdk-s3 = { version = "1.0" }
+aws-sdk-kms = { version = "1.0" }
+aws-config = { version = "1.0" }
 
 [features]
 default = ["remote"]
 remote = ["dep:reqwest"]
-fp16kernels = ["lance-linalg/fp16kernels"]
\ No newline at end of file
+fp16kernels = ["lance-linalg/fp16kernels"]
+integration-test = []
```

Comment:
Maybe `store-test` instead of `integration-test` since "integration test" is a pretty ambiguous term?